src/SymfonyCorp/Connect/OAuthConsumer.php

```diff
@@ -151,7 +166,13 @@ public function getEndpoint()
 
     public function getBrowser()
     {
-        return $this->browser;
+        @trigger_error(sprintf('"%s()" has been deprecated since 5.1, use "%s::getHttpClient()" instead.', __METHOD__, __CLASS__), E_USER_DEPRECATED);
```

Comment:
since symfony/connect - the instead part is telling about a method that doesn't exist, lets remove it